docs/rules-engine/how-to-use/debug.md

```diff
@@ -2,4 +2,39 @@
 
 For debugging, there is a dev tool service that you can activate in your application.
 It allows you to display a lot of different information about runs.
-Please have a look at [Dev Tools](../../dev-tools/chrome-devtools.md) for more details.
+
+## Enable Chrome extension debugging
```

Comment:
maybe we could centralize all the different setups in the `docs/dev-tools/chrome-devtools.md` file and point to it
mainly to avoid duplication

Reply:
I am not sure, the purpose is also to have this documentation per package and it makes sense to have a documentation dedicated to the setup of the debugging service for this specific package.
Of course we should also keep the link to the chrome-devtools generic documentation.
I am not sure we can considerer it as duplication when it is actually a dedicated doc